content/en/docs/getting-started/setup/create-cluster/google.md

```diff
@@ -7,8 +7,261 @@ publishdate: 2017-02-01
 lastmod: 2017-02-01
 weight: 50
 ---
-# Creating a Google cluster
 
+Google Cloud Platform (GCP) offers a powerful service for creating
+robust, highly-available clusters using the Kubernetes automated
```

Comment:
With our recommended setup it is not highly available. For that to be true, the cluster should be regional and not zonal.